File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextBorder.java
package ja.burhanrashid52.photoeditor;

public class TextBorder {
    float corner;
    int backGroundColor;
    int strokeWidth;
    int strokeColor;


    public TextBorder(float corner, int backGroundColor, int strokeWidth, int strokeColor) {
        this.corner = corner;
        this.backGroundColor = backGroundColor;
        this.strokeWidth = strokeWidth;
        this.strokeColor = strokeColor;
    }

    public float getCorner() {
        return corner;
    }

    public void setCorner(float corner) {
        this.corner = corner;
    }

    public int getBackGroundColor() {
        return backGroundColor;
    }

    public void setBackGroundColor(int backGroundColor) {
        this.backGroundColor = backGroundColor;
    }

    public int getStrokeWidth() {
        return strokeWidth;
    }

    public void setStrokeWidth(int strokeWidth) {
        this.strokeWidth = strokeWidth;
    }

    public int getStrokeColor() {
        return strokeColor;
    }

    public void setStrokeColor(int strokeColor) {
        this.strokeColor = strokeColor;
    }
}

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextBorder.kt
package ja.burhanrashid52.photoeditor

class TextBorder(private var corner: Float, private var backGroundColor: Int, private var strokeWidth: Int, private var strokeColor: Int) {
    fun getCorner(): Float {
        return corner
    }

    fun setCorner(corner: Float) {
        this.corner = corner
    }

    fun getBackGroundColor(): Int {
        return backGroundColor
    }

    fun setBackGroundColor(backGroundColor: Int) {
        this.backGroundColor = backGroundColor
    }

    fun getStrokeWidth(): Int {
        return strokeWidth
    }

    fun setStrokeWidth(strokeWidth: Int) {
        this.strokeWidth = strokeWidth
    }

    fun getStrokeColor(): Int {
        return strokeColor
    }

    fun setStrokeColor(strokeColor: Int) {
        this.strokeColor = strokeColor
    }
}